### ISSUE TITLE
NumberFormatException on inserting record with implicit attribute "id" set to uuid() as default

### ISSUE THREAD

**Issue by andreyvk** [orientechnologies/orientdb#6427]
Hi all,

- I have a parent class which has an attribute called `id` which is a string and default value is `uuid()`
- There's a child class `I18N` which extends this parent class
- When i try to insert a record into `I18N` studio throws a `java.lang.NumberFormatException`

I was able to reproduce it in my DB with whole schema seeded, but somehow it worked on  a test db  with only parent and child classes in place.

Here's a log btw:

```
2016-07-18 11:47:00:463 FINE  {db=OSystem} onClose [OLuceneIndexFactory]
2016-07-18 11:47:00:464 FINE  {db=nsdev} Read record #5:0 v.1 size=171 bytes [OLocalPaginatedStorage]
2016-07-18 11:47:00:465 FINE  {db=nsdev} Read record #4:0 v.2 size=69 bytes [OLocalPaginatedStorage]
2016-07-18 11:47:00:465 FINE  {db=nsdev} onOpen [OLuceneIndexFactory]
2016-07-18 11:47:00:466 FINE  {db=nsdev} onClose [OLuceneIndexFactory]Caught exception
com.orientechnologies.orient.core.exception.OCommandExecutionException: Error on execution of command: sql.insert into I18N set  name="test", language="test"
        DB name="nsdev"
        at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.executeCommand(OAbstractPaginatedStorage.java:2623)
        at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.command(OAbstractPaginatedStorage.java:2548)
        at com.orientechnologies.orient.core.command.OCommandRequestTextAbstract.execute(OCommandRequestTextAbstract.java:69)
        at com.orientechnologies.orient.server.network.protocol.http.command.post.OServerCommandPostCommand.execute(OServerCommandPostCommand.java:107)
        at com.orientechnologies.orient.graph.server.command.OServerCommandPostCommandGraph.execute(OServerCommandPostCommandGraph.java:37)
        at com.orientechnologies.orient.server.network.protocol.http.ONetworkProtocolHttpAbstract.service(ONetworkProtocolHttpAbstract.java:165)
        at com.orientechnologies.orient.server.network.protocol.http.ONetworkProtocolHttpAbstract.execute(ONetworkProtocolHttpAbstract.java:613)
        at com.orientechnologies.common.thread.OSoftThread.run(OSoftThread.java:77)
Caused by: java.lang.NumberFormatException: For input string: "84102b55-8218-4e5b-be98-9ae398b7c727"
        at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
        at java.lang.Integer.parseInt(Integer.java:580)
        at java.lang.Byte.parseByte(Byte.java:149)
        at java.lang.Byte.parseByte(Byte.java:175)
        at com.orientechnologies.orient.core.sql.OSQLHelper.parseStringNumber(OSQLHelper.java:200)
        at com.orientechnologies.orient.core.sql.OSQLHelper.parseValue(OSQLHelper.java:179)
        at com.orientechnologies.orient.core.sql.OSQLHelper.parseDefaultValue(OSQLHelper.java:70)
        at com.orientechnologies.orient.core.record.impl.ODocument.convertFieldsToClass(ODocument.java:2643)
        at com.orientechnologies.orient.core.record.impl.ODocument.setClassName(ODocument.java:2004)
        at com.orientechnologies.orient.core.record.impl.ODocument.<init>(ODocument.java:161)
        at com.orientechnologies.orient.core.sql.OCommandExecutorSQLInsert.execute(OCommandExecutorSQLInsert.java:228)
        at com.orientechnologies.orient.core.sql.OCommandExecutorSQLDelegate.execute(OCommandExecutorSQLDelegate.java:72)
        at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.executeCommand(OAbstractPaginatedStorage.java:2602)
        ... 7 more

2016-07-18 11:47:00:469 SEVER Internal server error:
java.lang.NumberFormatException: For input string: "84102b55-8218-4e5b-be98-9ae398b7c727" [ONetworkProtocolHttpDb]
2016-07-18 11:47:03:567 FINE  [ONetworkProtocolHttpAbstract.execute] Requested: GET /listDatabases [ONetworkProtocolHttpDb]
2016-07-18 11:47:03:568 FINE  {db=OSystem} onOpen [OLuceneIndexFactory]

```

### Running Mode
- [ ] Embedded, using PLOCAL access mode
- [ ] Embedded, using MEMORY access mode
- [x] Remote

### Misc
- [ ] I have a distributed setup with multiple servers. How many? 
- [ ] I'm using the Enterprise Edition

### OrientDB Version
- [ ] v2.0.x - Please specify last number: 
- [ ] v2.1.x - Please specify last number: 
- [x] v2.2.x - Please specify last number: 

### Operating System
- [x] Linux
- [ ] MacOSX
- [ ] Windows
- [ ] Other Unix
- [ ] Other, name? 

### Java Version
- [ ] 6
- [ ] 7
- [x] 8

**Comment by andreyvk** [comment]
For more info, we were moving schema from 2.1 to 2.2 today. That's when this happened

**Comment by luigidellaquila** [comment]
Hi @andreyvk 

I just pushed a fix for this problem. The fix will be released with 2.2.6

Thanks

Luigi

**Comment by andreyvk** [comment]
Thanks @luigidellaquila !
